src/engraving/dom/lyricsline.cpp

```diff
@@ -40,24 +40,28 @@ namespace mu::engraving {
 //   LyricsLine
 //---------------------------------------------------------
 
+static const ElementStyle lyricsLineElementStyle {
+    { Sid::lyricsDashLineThickness, Pid::LINE_WIDTH }
+};
+
 LyricsLine::LyricsLine(EngravingItem* parent)
-    : SLine(ElementType::LYRICSLINE, parent, ElementFlag::NOT_SELECTABLE)
+    : SLine(ElementType::LYRICSLINE, parent)
 {
-    setGenerated(true);             // no need to save it, as it can be re-generated
     setDiagonal(false);
-    setLineWidth(style().styleS(Sid::lyricsDashLineThickness));
+    initElementStyle(&lyricsLineElementStyle);
     setAnchor(Spanner::Anchor::SEGMENT);
     m_nextLyrics = 0;
+    setGenerated(true);             // no need to save it, as it can be re-generated
```

Comment:
GENERATED is also a flag I believe, so you could add the flag to the constructor's arguments instead of setting it here